### DISCUSSION TITLE
How to skip msi build for windows?

### DISCUSSION THREAD

**Discussion by HuakunShen** [tauri-apps/tauri#10900] [Q&A] [answered]
Trying to use version like 0.1.0-beta.0, but will fail for msi build.

Is it possible to skip msi build completely and upload only exe to release?

Here is a sample run of my application
https://github.com/kunkunsh/kunkun/actions/runs/10707529761/job/29687718096#step:15:1166

**Comment by FabianLars** [comment, marked as answer]
You can change this https://github.com/kunkunsh/kunkun/blob/dev/apps/desktop/src-tauri/tauri.conf.json#L39 from `"all"` to an array of targets you want. So if you want "all" without msi it would currently be this: `[“deb”, “rpm”, “appimage”, “nsis”, “app”, “dmg”]`.

Alternatively you can change your workflow file here https://github.com/kunkunsh/kunkun/actions/runs/10707529761/workflow#L28 from `'--verbose'` to `'--verbose -b nsis'`

**Reply by HuakunShen** [reply]
Thank you!

**Reply by MC-XiaoHei** [reply]
because of the branch name changed, the file is here now

https://github.com/kunkunsh/kunkun/blob/develop/apps/desktop/src-tauri/tauri.conf.json#L45

**Reply by HuakunShen** [reply]
> because of the branch name changed, the file is here now
> 
> https://github.com/kunkunsh/kunkun/blob/develop/apps/desktop/src-tauri/tauri.conf.json

Thanks for clarifying

**Comment by ayangweb** [comment]
@FabianLars Hey 👋, excuse me, I wrote it this way, I can achieve the effect, but the signature file is not generated, why is that?

https://github.com/ayangweb/ayangweb-EcoPaste/blob/55563981380c0b5242736f11dc4de052bf9e973c/.github/workflows/release.yml#L107

<img width="1198" alt="image" src="https://github.com/user-attachments/assets/1ba3aea6-8f0f-441e-b275-cc07623c3fe0">
